src/wasm-interpreter.h

```diff
@@ -1917,6 +1917,15 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     if (!data) {
       trap("null ref");
     }
+
+    // This is only correct if all the bytes stored in `values` correspond to
+    // single unicode code points. See `visitStringWTF16Get` for details.
+    for (Index i = 0; i < data->values.size(); ++i) {
+      if (uint32_t(data->values[i].geti32()) > 127) {
+        return Flow(NONCONSTANT_FLOW);
+      }
+    }
```

Comment:
Perhaps it makes sense to add a helper for this? Could be a templated function in `src/support/string.h`, or maybe a helper in this file as part of the interpreter would be better, I'm not sure.

Reply:
Since we're operating on lists of literals, I've added a helper to the interpreter.